package.json

```diff
@@ -34,7 +34,7 @@
     "typescript": "^4.9.4"
   },
   "scripts": {
-    "compile-wasm": "turbo run compile",
+    "compile-wasm": "pnpm --filter @anonklub/merkle-tree compile",
```

Comment:
I see, I agree that this looks better but the only drawback is that you will need to have turbo installed globally. And that is not a good practice, its okay for short-term but for long-term we need a better way I think. 
That's why thought of separating the Rust wasm compilation from the turbo stuff so that I can easily do `pnpm install`.

Reply:
> you will need to have turbo installed globally

You don't. I don't have turbo installed globally and this script is working fine for me